src/react/hooks/__tests__/useBackgroundQuery.test.tsx

```diff
@@ -3569,14 +3568,23 @@ describe("useBackgroundQuery", () => {
         const [queryRef, { refetch }] = useBackgroundQuery(query, {
           variables: { id },
         });
-        return <Todo refetch={refetch} queryRef={queryRef} onChange={setId} />;
+        const onRefetchHandler = () => {
+          refetch();
+        };
+        return (
+          <Todo
+            refetchHandler={onRefetchHandler}
+            queryRef={queryRef}
+            onChange={setId}
+          />
+        );
       }
 
       function Todo({
         queryRef,
-        refetch,
+        refetchHandler,
       }: {
-        refetch: RefetchFunction<Data, OperationVariables>;
+        refetchHandler: () => void;
```

Comment:
```suggestion
        onRefetch: () => void;
```

See my other comment and let me know if you like this :)